platform/linux-generic/arch/default/odp_atomic.h

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2021, Arm Limited
```

Comment:
In other files ARM is written with all caps.

Reply:
I will change it to be consistent with the other files. Thanks!